mlir/utils/jenkins/Jenkinsfile

```diff
@@ -29,6 +29,22 @@ void getAndBuildMIOpen(String prefixOpt, String cmakeOpts) {
     buildMIOpen(cmakeOpts)
 }
 
+void buildMIGraphX(String cmakeOpts) {
+    sh '[ ! -d build ] || rm -rf build'
+    cmakeBuild generator: 'Unix Makefiles',\
+        buildDir: 'build',\
+        buildType: 'Release',\
+        installation: 'InSearchPath',\
+        cmakeArgs: ${cmakeOpts}
+    sh 'cd build; make -j $(nproc)'
+}
+
+void getAndBuildMIGraphX(String cmakeOpts) {
+    git branch: 'be70702dc80d59126d6b4011d6cc7913193db566', poll: false,\
```

Comment:
branch `develop`

Reply:
do we want a moving target ? I think that is fine for weekly/nightly to see when it brakes -- but I'd like to stop PRs getting merged that breaks that we know to work.

Reply:
PR CI's already pretty long, and I don't want to add another build and a rather substantial validation sequence to it.

Reply:
I'd like to see this running nightly or weekly and also a flag to allow us to force testing it on PRs where we're concerned.

Reply:
And yes, the point is to have a moving target, though we can have a `migraphx-branch` parameter analogous to miopen-branch for when we've got a pending PR

Reply:
Re: PR CI, fair enough. Lets revisit if we have nightly or weekly breakages then?

Reply:
(long is subjective.. I suppose but I like this project's faster CI ... So keep it that way)